### PR TITLE
Warm reboot: warm_restart config and show commands

### PR DESCRIPTION
**Pull request by jipanyang**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

**- What I did**
warm_restart config and show commands

**- How I did it**

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

```

root@sonic:/# config warm_restart enable
root@sonic:/# config warm_restart disable swss
root@sonic:/# 


root@sonic:/# show warm_restart 
Usage: show warm_restart [OPTIONS] COMMAND [ARGS]...

  Show warm restart configuration and state

Options:
  -?, -h, --help  Show this message and exit.

Commands:
  config  Show warm restart config
  state   Show warm restart state


root@sonic:/# show warm_restart  config
name    enable    timer_name    timer_duration
------  --------  ------------  ----------------
swss    false     NULL          NULL
system  true      NULL          NULL
root@sonic:/# 
root@sonic:/# config warm_restart enable swss
root@sonic:/# 


root@sonic:/# show warm_restart config
name    enable    timer_name    timer_duration
------  --------  ------------  ----------------
swss    true      NULL          NULL
system  true      NULL          NULL
root@sonic:/# 


root@sonic:/# show warm_restart  state
name          restart_count  state
----------  ---------------  -------
orchagent                 2  init
neighsyncd                1
portsyncd                 2
vlanmgrd                  1

root@sonic:/# show warm_restart state 
name          restart_count  state
----------  ---------------  ----------
orchagent                 2  reconciled
neighsyncd                2
portsyncd                 2
vlanmgrd                  2


```

-->

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -1170,6 +1171,91 @@ def line():
     """Show all /dev/ttyUSB lines and their info"""
     cmd = "consutil show"
     run_command(cmd, display_cmd=verbose)
+    # TODO: Stub
```
Comment (zhenggen-xu) [inline show/main.py]: Are you trying to add something here or just by accident?

File: show/main.py [hunk 2]
```diff
@@ -1170,6 +1171,91 @@ def line():
...
+                r.append(data[k]['timer_duration'])
+            else:
+                r.append("NULL")
+                r.append("NULL")
+
+            table.append(r)
+
+        return table
+
+    header = ['name', 'enable', 'timer_name', 'timer_duration']
```
Comment (zhenggen-xu) [inline show/main.py]: header should be adjusted to the new schema.

**Comment by jipanyang** [comment]
```
root@sonic:/# config warm_restart neighsyncd_timer 300
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    false     neighsyncd_timer  300
system  true      NULL              NULL
root@sonic:/# show warm_restart state
name          restart_count  state
----------  ---------------  ----------
neighsyncd                1
orchagent                 1  reconciled
vlanmgrd                  1
portsyncd                 1
root@sonic:/# config warm_restart enable swss
root@sonic:/# 
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name    timer_duration
------  --------  ------------  ----------------
swss    true      NULL          NULL
system  true      NULL          NULL
root@sonic:/# config warm_restart neighsyncd_timer 300
root@sonic:/# config warm_restart enable swss
```
```
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  300
system  true      NULL              NULL
root@sonic:/# config warm_restart neighsyncd_timer 200
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  200
system  true      NULL              NULL
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name    timer_duration
------  --------  ------------  ----------------
system  true      NULL          NULL
root@sonic:/# config warm_restart neighsyncd_timer 200
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    false     neighsyncd_timer  200
system  true      NULL              NULL
root@sonic:/# config warm_restart enable swss
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  200
system  true      NULL              NULL
root@sonic:/# 
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  200
system  true      NULL              NULL
root@sonic:/# config warm_restart disable swss
root@sonic:/# 
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    false     neighsyncd_timer  200
system  true      NULL              NULL
```

```
root@sonic:/# config warm_restart enable  swss
root@sonic:/# 
root@sonic:/# 
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  200
system  true      NULL              NULL
root@sonic:/# 
root@sonic:/# 
root@sonic:/# config warm_restart 
disable           enable            neighsyncd_timer  
root@sonic:/# config warm_restart neighsyncd_timer     
root@sonic:/# config warm_restart neighsyncd_timer 300
root@sonic:/# 
root@sonic:/# 
root@sonic:/# show warm_restart config
name    enable    timer_name        timer_duration
------  --------  ----------------  ----------------
swss    true      neighsyncd_timer  300
system  true      NULL              NULL
root@sonic:/# 
root@sonic:/# 
root@sonic:/# config warm_restart neighsyncd_timer  0
neighsyncd warm restart timer must be in range 1-9999
Aborted!
root@sonic:/# config warm_restart neighsyncd_timer  10000
neighsyncd warm restart timer must be in range 1-9999
Aborted!
root@sonic:/# 

```

File: show/main.py (hunk 2 above)
Comment (zhenggen-xu) [reply show/main.py]: How do we present multiple timers at CLI?  Would it make more sense to have the real timer name at the header?

Of course, it is up to people who work on other timers to add the CLI later.

Comment (jipanyang) [reply show/main.py]: I don't think putting fixed timer name in header is a good idea.
If there are multiple timers per docker, just fill space in other fields and make multiple rows. 
Anyway it is kind of UI things,  people may have different opinions.

Comment (zhenggen-xu) [reply show/main.py]: OK,  it is fine as is as baseline.

**Pull request merged**